packages/components/src/tabs/index.tsx

```diff
@@ -42,10 +42,20 @@ function Tabs( {
 		selectedId: selectedTabId && `${ instanceId }-${ selectedTabId }`,
 	} );
 
-	const isControlled = selectedTabId !== undefined;
+	const isControlled = useMemo(
+		() => selectedTabId !== undefined,
+		[ selectedTabId ]
+	);
```

Comment:
What's the reason for memoizing this variable?

Reply:
I think that was leftover from something else I'd been experimenting with, thanks for spotting it.